tests/httpfake.h

```diff
@@ -134,6 +134,9 @@ class HttpFake : public HttpInterface {
         devices_count++;
         EXPECT_EQ(data["deviceId"].asString(), "tst149_device_id");
         return HttpResponse(Utils::readFile("tests/test_data/cred.p12"), 200, CURLE_OK, "");
+      } else if (url == "tst149/events") {
+        ++events_seen;
+        return HttpResponse("", 200, CURLE_OK, "");
```

Comment:
The test149 section is only used by the `ProvisionOnServer` test in uptane_test. It might be worth adding checks in that test that `events_seen` has the expected values as we check for the other event counts.